### PR TITLE
Add logo and capitalise Reek’s name

### PR DESCRIPTION
**Pull request by chastell**
Add the reek logo from the wonderful @SonjaHeinen. :sparkling_heart:

(It turns out GitHub does not serve SVG files with the proper `Content-Type`, so we have to embed a PNG instead.)

### CODE REVIEW

**Comment by mvz** [comment]
Maybe we can move the images to a subdirectory? Also, can you add the b/w versions?

**Comment by troessner** [comment]
-> https://github.com/troessner/reek/blob/logo/README.md

Without "reek" on the right side this looks a little strange....

**Comment by chastell** [comment]
> Maybe we can move the images to a subdirectory? Also, can you add the b/w versions?

Ah, good points! I will.

> Without "reek" on the right side this looks a little strange....

Yeah, this is a WIP until we have the layout figured out.

**Comment by chastell** [comment]
This now adds Reek’s logo to the repo, puts it in README and spells Reek’s name as ‘Reek’ where it means the project / tool and as `reek` where it means the CLI executable (which turns out as a nice, clarifying distinction).

**Comment by mvz** [comment]
I'd like to see:
* PNG version of the logo without the name
* SVG version of the logo with the name
* PNG version of the b/w logo
* SVG version of the b/w logo

Apart from that: Awesome work on capitalizing all the names! :tada:

File: README.md [hunk 1]
```diff
@@ -1,4 +1,6 @@
-# `reek`: code smell detection for Ruby
+![reek logo](logo/reek.text.png)
```
Comment (troessner) [inline README.md]: Hmmm, no offense, but the way it looks right now I thought it was a layout bug...
How about scaling the graphic down and align the "Reek" literal vertically to the ruby?

**Comment by chastell** [comment]
@mvz Good catches, I’ll add them once we have the final version approved by everyone. :)

@troessner I’m not sure what you mean by ‘align vertically’. The ruby gem is much ‘heavier’ at its top and the text at its bottom, so when you align the vertical centres of the elements it doesn’t look aligned at all:

reek text ubuntu centred 1

Aligning the centres of the lowercase letters with the gem’s ‘equator’ doesn’t work either:

reek text ubuntu centred 2

How about making the text bigger and aligning the tops of the lowercase letters with the gem’s ‘equator’?

reek text ubuntu centred 4

(I’m happy to work on this more; I’ve never designed anything like this, so I have zero experience, but I have a lot of patience and want all of us to be happy.) :)

**Comment by chastell** [comment]
…or we can make the logo size match the text, but I really like it and it seems to diminish it quite a bit:

reek text ubuntu centred 5

**Comment by mvz** [comment]
@chastell Of the four alternatives you've presented, I like the first one (basically center alignment) and the third one (slightly bigger text). The one with huge text looks out of balance.

**Comment by troessner** [comment]
Yes, i'm also voting for the first one like @mvz !

**Comment by chastell** [comment]
Done! Let me know what you think. :)

**Comment by mvz** [comment]
> It turns out GitHub does not serve SVG files with the proper Content-Type, so we have to embed a PNG instead.

Ha! How weird that it will embed SVG files just fine, *except* when they're part of the repo :crying_cat_face:.

**Comment by mvz** [comment]
I think it looks good. LGTM!

**Comment by troessner** [comment]
Merging this bad boy!

**Comment by troessner** [comment]
Awesome, btw, thanks again @@SonjaHeinen!

**Comment by SonjaHeinen** [comment]
High Five!

2015-10-17 23:05 GMT+02:00 Timo Rößner <notifications@github.com>:

> Merged #737 <https://github.com/troessner/reek/pull/737>.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/troessner/reek/pull/737#event-438322479>.
>



-- 
Sonja Heinen

+49 176 6262 3734
sonja.heinen@gmail.com

**Pull request merged**